qt-models/cylindermodel.cpp

```diff
@@ -237,6 +239,15 @@ QVariant CylindersModel::data(const QModelIndex &index, int role) const
 			return static_cast<int>(cyl->type.workingpressure.mbar);
 		case SIZE_INT:
 			return static_cast<int>(cyl->type.size.mliter);
+		case SENSORS: {
+			QStringList sensors;
+			auto *sample = current_dc->sample;
+			for (int s = 0; sample && s < MAX_SENSORS; ++s) {
```

Comment:
So i realized that just looking at the first sample doesn't work in all cases. It'll probably be correct for unedited dives, but when the dive is saved only the first sample with pressure data will save the sensor information, so when loading a dive like that the display will be wrong.